api/app/Models/Tag.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: redjik
+ * Date: 13.07.15
+ * Time: 15:07
+ */
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class Tag extends BaseModel
+{
+    public $table = 'tags';
+
+    protected $primaryKey = 'tag_id';
+
+    public $incrementing = true;
```

Comment:
This surrogate key should be a uuid. In future we're going to implement internationalisation with translations, and it is simpler if we can have translation keys being unique ids across all tables.